services/azure-devops/azure-devops-build.service.js

```diff
@@ -6,30 +6,25 @@ const { keywords, fetch, render } = require('./azure-devops-helpers')
 
 const documentation = `
 <p>
```

Comment:
As discussed in the PR, I edited this down to the essentials. Probably we will want to do the same with some of the others in follow-on work.

Reply:
This looks good! And also agreed on doing the same in some future work